spec/02-integration/14-tracing/03-tracer-pdk_spec.lua

```diff
@@ -77,5 +85,39 @@ for _, strategy in helpers.each_strategy() do
         end
       end)
     end)
+
+    describe("spans start/end times are consistent with their hierarchy", function ()
+      lazy_setup(function()
+        setup_instrumentations("all", false, 1)
+      end)
+
+      lazy_teardown(function()
+        helpers.stop_kong()
+      end)
+
+      it("sets child lifespan within parent's lifespan", function ()
+        for _ = 1, 100 do
```

Comment:
How long does this test take... How about we do it with a mocked unit test?

Reply:
this test takes 0.7 seconds to execute in my local machine, let's see how long it takes on CI once the build succeeds, but I don't think it will exceed 1 second

Reply:
Then I think it's fine.